src/Components/Publishing/Display/__test__/DisplayPanel.test.tsx

```diff
@@ -1,10 +1,15 @@
 import "jest-styled-components"
 import React from "react"
 import renderer from "react-test-renderer"
-import { Campaign, UnitPanel } from "../../Fixtures/Components"
+import { Campaign, UnitPanel, UnitPanelVideo } from "../../Fixtures/Components"
 import { DisplayPanel } from "../DisplayPanel"
 
-it("renders the display panel", () => {
+it("renders the display panel with an image", () => {
   const displayPanel = renderer.create(<DisplayPanel unit={UnitPanel} campaign={Campaign} />).toJSON()
   expect(displayPanel).toMatchSnapshot()
 })
+
+it("renders the display panel with video", () => {
+  const displayPanel = renderer.create(<DisplayPanel unit={UnitPanelVideo} campaign={Campaign} />).toJSON()
+  expect(displayPanel).toMatchSnapshot()
+})
```

Comment:
Might be helpful to start integrating some unit tests along with snapshot tests, especially for interactive components.

Reply:
Yeah was looking at testing for this, but had trouble getting access to the video ref -- if you're up to pair today I'd be glad to have help figuring this out.